perpVects/MakefileDev

```diff
@@ -1,12 +1,12 @@
 #CC := clang++
-CC := g++
+CC := g++-4.9
```

Comment:
Is the move to g++-4.9 just for testing the CUDA stuff?  For the normal runs with non-CUDA, we will be using a much later g++, right?  Please explain this change.

I understand the move to C++11 since we need to write for the lowest-common-version.